src/components/carousel/_index.scss

```diff
@@ -0,0 +1,165 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
```

Comment:
We should not remove the `outline` by default - this break accessibility, more so when we have not provided a replacement option which we would then also need to ensure is accessible

Reply:
Definitely, removed